client/public/index.html

```diff
@@ -4,13 +4,14 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/LogoGlyphMd.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+    <meta name="theme-color" media="(prefers-color-scheme: light)" content="white">
+    <meta name="theme-color" media="(prefers-color-scheme: dark)" content="black">
+    <title>AskGov</title>
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="The Q&A Platform for Government Related Services"
```

Comment:
Can we stick to the tagline that we first had?
```suggestion
      content="Official answers from the Singapore Government"
```